dev-lang/retro/retro-2021.4.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Clean, elegant, and pragmatic dialect of Forth"
+DESCRIPTION="Retro is a clean, elegant, and pragmatic dialect of Forth. \
+It provides a simple alternative for those willing to make a break from legacy \
+systems.
+
+The language draws influences from many sources including traditional \
+Forth systems, cmForth, colorForth, Factor, and Parable. It was \
+designed to be easy to grasp and adapt to specific uses."
+HOMEPAGE="https://forthworks.com/retro/"
+COPYRIGHT="2008-2021, Charles Childers"
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="https://forthworks.com/retro/r/RETRO12-$portVersion.tar.gz"
+CHECKSUM_SHA256="9d43aae72527e7b18fdbd3b4237d425b79ad95bb3d0dee6655264970478524b0"
+SOURCE_DIR="RETRO12-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	retro$secondaryArchSuffix = $portVersion
+	cmd:retro$secondaryArchSuffix = $portVersion
```

Comment:
if retro is actually installed in $prefix/bin, you can drop $secondaryArchSuffix on all cmd:*. I suppose you didn't test on x86 yet.

Reply:
fixed